src/visualizations/graph_2d_visuals.py

```
@@ -129,49 +130,53 @@ def visualize_2d_graph(graph: AssetRelationshipGraph,
     return positions
 
-def _create_spring_layout_2d(positions_3d: Dict[str, Tuple[float, float, float]], 
+
+def _create_spring_layout_2d(positions_3d: Dict[str, Tuple[float, float, float]],
                             asset_ids: List[str]) -> Dict[str, Tuple[float, float]]:
     """Convert 3D positions to 2D using projection"""
     positions = {}
-    
+
     for asset_id in asset_ids:
         x3d, y3d, z3d = positions_3d[asset_id]
```

Ruff (0.14.2)

Unpacked variable `z3d` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)